### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by ghost** [community/community#106554] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @Ninja0930  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be closed and locked to keep our community nice and tidy 🧹.
